thoth/user_api/api_v1.py

```diff
@@ -703,12 +703,28 @@ def schedule_kebechet(body: dict):
 
 
 def schedule_kebechet_webhook(body: typing.Dict[str, typing.Any]):
-    """Schedule Kebechet on Openshift."""
+    """Schedule Kebechet run-webhook on Openshift using Argo Workflow."""
+    payload, webhook_payload = {}, {}
+    headers = connexion.request.headers
+    
+    if "X-GitHub-Event" in headers:
+        webhook_payload["event"] = headers["X-GitHub-Event"]
+        webhook_payload["payload"] = body
+        webhook_payload["service"] = "github"
+    elif "X_GitLab_Event" in headers:
+        webhook_payload["payload"] = body
```

Comment:
I'm not familiar with the actual payload content here - can all the fields be kept consistent (in case of any service, `payload`, `event` and `service` keys will be always present)?

Reply:
I have added it, even though we use object kind inside the payload for gitlab.